web/src/app/admin/admin-routing.module.ts

```diff
@@ -12,6 +13,11 @@ const routes: Routes = [
     component: UsersListComponent,
     canActivate: [AdminGuard],
   },
+  {
+    path: 'configure-webhooks',
```

Comment:
Can the admin paths have `admin` before, so it would be

- `admin/users`
- `admin/webhooks`

Reply:
okay